DashAI/back/dataloaders/classes/dataloader.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
+
+from datasets import ClassLabel
+
+from DashAI.back.config_object import ConfigObject
+
+
+class DataLoader(ConfigObject):
+    """
+    Abstract class with base methods for all data loaders
+    """
+
+    @abstractmethod
+    def load_data(self, dataset_path, url=None):
+        pass
+
+    @abstractmethod
+    def set_task_format(self, task):
+        pass
+
+    def split_dataset(self, dataset, params, class_column):
+        """
+        Split the dataset in train, test and validation data
+        """
+        stratify_column = None
```

Comment:
I will add this checks, I had not considered it before because I assumed that the dictionary would come with the correct data, Which in fact may not always be the case.